test/parallel/test-stream-readable-async-iterators.js

```diff
@@ -396,6 +396,28 @@ async function tests() {
     }
   }
 
+  console.log('readable side of a transform stream pushes null');
+  {
+    const transform = new Transform({
+      objectMode: true,
+      transform(chunk, enc, cb) {
```

Comment:
This is never called, right? Can you use `transform: common.mustNotCall()` in that case (for clarity in the test)?

Reply:
:+1: - done